src/org/zalando/stups/essentials/core.clj

```diff
@@ -40,16 +40,28 @@
 (def default-controller-config
   {:api-allowed-realms "services,employees"})
 
+(defn config-true?
+  "Determine if a config element reflects truth in a boolean context."
+  [setting]
+  (some? (#{"true" "1" true 1} setting)))
```

Comment:
I just use `some?` to make sure that the result is a Boolean; that way I can use `nil` to mean "the setting is not defined".